selfdrive/controls/lib/vehicle_model.py

```diff
@@ -19,6 +19,9 @@
 
 from cereal import car
 
+ACCELERATION_DUE_TO_GRAVITY = 9.8
```

Comment:
Is this accurate enough? Isn't it 9.81, and also altitude dependent?

Reply:
![image](https://user-images.githubusercontent.com/96022003/146541970-fbdaef1a-cc28-4316-8f67-070cfea9be90.png)
It varies by more than the error, and the relative error between 9.81 and 9.8 is 0.1%.
Going from 6400km from the center of the earth at sea level to 6403km (+10k feet) also drops the acceleration by less than 0.1%